components/Event/EventInfo.js

```diff
@@ -0,0 +1,67 @@
+import { DateTime } from "luxon";
```

Comment:
Is this dependency really necessary? Isn't [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) enough?

Reply:
I guess for now it is :disappointed: